### ISSUE TITLE
deviations can now have multiple images

### ISSUE THREAD

**Issue by Yay295** [wix-incubator/DeviantArt-API#256]
https://www.deviantart.com/team/journal/Extend-your-creativity-with-multi-image-deviations-1127107538
https://www.deviantart.com/topro/art/Fluffy-Like-A-Cloud-Llama-1121307720

This affects some of the `/deviation/` endpoints.

**Comment by SomewhatDamaged** [comment]
Yeah. I only see one deviation_ID in the page `<HEAD>`, and when that deviation_ID is called from `/oauth2/deviation/<deviation_ID>` I only get the first image.

There are probably other endpoints that will need altering, but that and `/oauth2/deviation/metadata/<deviation_ID>` are the main two I hit, and the latter will likely be fine.

**Comment by NoLSP** [comment]
The ability to add multiple images appeared more than a month ago, but nothing has changed in the API. Please tell me, is there any work on api edits?

**Comment by SomewhatDamaged** [comment]
Crickets chirping, tumbleweed blowing by.

I think it's dead. This feature has been out for ~6 months, and still no way to programmatically detect or access it.

**Comment by Yay295** [comment]
The only way I've found is to get the page HTML and look for an element matching `main .reset-button[aria-label="Click to view images by scrolling through them"]`. I don't know of a way to get any information about the other images though.

**Comment by SomewhatDamaged** [comment]
Now close to 8 months, and still no support. It'd be nice if you at least triaged this, DA!

**Comment by SomewhatDamaged** [comment]
10 month mark is a few days away. No support for this feature still.

**Comment by SomewhatDamaged** [comment]
20 months

It's dead, Jim. Staff don't care. AI bros don't care. DA is just a slop factory now.

> This place is a message... and part of a system of messages... pay attention to it!
> 
> Sending this message was important to us. We considered ourselves to be a powerful culture.
> 
> This place is not a place of honor... no highly esteemed deed is commemorated here... nothing valued is here.
> 
> What is here was dangerous and repulsive to us. This message is a warning about danger.
> 
> The danger is in a particular location... it increases towards a center... the center of danger is here... of a particular size and shape, and below us.
> 
> The danger is still present, in your time, as it was in ours.
> 
> The danger is to the body, and it can kill.
> 
> The form of the danger is an emanation of energy.
> 
> The danger is unleashed only if you substantially disturb this place physically. This place is best shunned and left uninhabited.
